universal/algos/olmar.py

```diff
@@ -67,7 +72,31 @@ def update(self, b, x, eps):
         # project it onto simplex
         return tools.simplex_proj(b)
 
+    def update_tco(self, b, x_pred):
+        """
+        Transaction Costs Optimization
+        Paper : https://ink.library.smu.edu.sg/cgi/viewcontent.cgi?referer=&httpsredir=1&article=4761&context=sis_research
+        """
+
+        trx_fee_pct = 0.1
```

Comment:
Could you add this to `__init__` as parameter? You can then reference it with `self.trx_fee_pct` here. Also please only run `update_tco` if `trx_fee_pct` is not `None` (which should be probably default value)